packages/assets/src/loader/parsers/textures/loadSVG.ts

```diff
@@ -0,0 +1,62 @@
+import type { ISVGResourceOptions, Texture } from '@pixi/core';
+import { BaseTexture, ExtensionType, SVGResource } from '@pixi/core';
+import { settings } from '@pixi/settings';
+import { getResolutionOfUrl } from '@pixi/utils';
+import { extname } from '../../../utils';
+import type { Loader } from '../../Loader';
+import type { LoadAsset } from '../../types';
+
+import type { LoaderParser } from '../LoaderParser';
+import { loadTextures } from './loadTexture';
+import { createTexture } from './utils/createTexture';
+
+export type SVGTextureData = {
+    baseTexture: BaseTexture;
+    svgResource: ISVGResourceOptions;
+};
```

Comment:
I don't understand the `baseTexture`, shouldn't this be `IBaseTextureOptions`? What about this instead since IBaseTextureOptions contains `resourceOptions`

```suggestion
export type SVGTextureData = IBaseTextureOptions;
```

Reply:
yeah you are correct. I've updated it!